modules/core/client/views/sidebar.client.view.html

```diff
@@ -17,27 +17,78 @@
                 <li>Current Study</li>
                 <li>
                     <button type="button" class="btn btn-transparent btn-block" data-toggle="button" ui-sref="home">
-                        <span class="glyphicon glyphicon-menu-hamburger"></span>Dashboard
+                        <div class="row">
+                            <div class="col-xs-1 col-sm-1 col-md-1 col-lg-1">
+                                <span class="glyphicon glyphicon-menu-hamburger"></span>
+                            </div>
+                            <div class="col-xs-10 col-sm-10 col-md-10 col-lg-10">
+                                <span style="float:left;">Dashboard</span>
```

Comment:
!fix For bootstrap, float can be used as pull, do a class="pull-left" for each of the times you used float